lib/rialto/etl/configs/stanford_organizations_to_sparql_statements.rb

```diff
@@ -10,7 +10,7 @@
 extend Rialto::Etl::NamedGraphs
 extend Rialto::Etl::Vocabs
 
-def contextualized_organization_name(organization)
+def contextualized_org_name(organization)
```

Comment:
Ha!

Reply:
Changed because Rubocop (within RubyMine) was complaining about the method name length. (I personally disagree.)